### PR TITLE
@svempolin & @louise-becker Moviebox

### PR DESCRIPTION
**Pull request by louise-becker**
Link to my deployed project:  https://moviebox-by-ll.netlify.app/

### CODE REVIEW

**Comment by Svempolin** [comment]
This is our Movieapp. We are proud that we made the loaders and the smal img.

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
+    fetch(BASE_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        setTimeout(() => {
+          setLoading(false);
+          setList(data.results);
+        }, 2000);
+      });
+  }, []);
+  // console.log(list)
```
Comment (HelenaW86) [inline code/src/App.js:25]: delete the console before pushing

File: code/src/components/Details/Details.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from "react";
+import { useParams, useHistory } from "react-router-dom";
+import { DETAILS_URL } from "../../utils/urls";
```
Comment (HelenaW86) [inline code/src/components/Details/Details.js:3]: shall it be ../ two times?

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,41 @@
...
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      <BrowserRouter>
+        <Header />
+        <Switch>
+          {loading && <Loading />}
```
Comment (JuliaNiki) [inline code/src/App.js:32]: Great that you've done a red level loading, good job!

File: code/src/components/Details/Details.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+.back-button {
+  z-index: 1;
+  position: absolute;
+  margin-left: 20px;
+  padding: 10px;
+  padding: 10px;
+  background: none;
+  border: none;
+  color: #fff;
+  font-weight: bolder;
```
Comment (HelenaW86) [inline code/src/components/Details/Details.css:24]: does bolder do the same as bold?

File: README.md [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
+In this project we got to use React.js, HTML, CSS, JSX and React components, API s in React - combining useState with useEffect. We also passed information such as movie titles in the url and picked this up in React router to load dynamic content.
+For functionality we followed a step by step lecture, and for the design we looked at an existing page and mimicked that one. 
+We started the project by making sure we could catch the correct data needed from the API. Then we went on with adding functionality with the help of React router, we also made a redirect when there was an error in the url request. Then we added styling to our page, for mobile and desktop.
+If we had more time we would maybe experiment more with styling of the page in mobile. 
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Please check out our page online here: XXXX.netlify.com
```
Comment (JuliaNiki) [inline README.md:15]: Please attach the link to you page :)

**Review by JuliaNiki — APPROVED** [review]
Great job :)

**Review by HelenaW86 — COMMENTED** [review]
Nice work! you have an error page for wrong path and a nice looking loader so the red level is complited! there is some commented out code that could be cleaned out before pushed. 
God job!

**Pull request closed without merging**